### PR TITLE
ENG-6571-added FPJS endpoint environment url

### PR DESCRIPTION
**Pull request by priya-xavier**
Currently only the PROD endpoint is going to be available in the sdk. The dev endpoint is only for internal testing.

### CODE REVIEW

File: NeuroID/src/advancedDeviceAndroidLib/java/com/neuroid/tracker/extensions/AdvancedDeviceExtension.kt [hunk 1]
```diff
@@ -25,13 +26,18 @@ fun NeuroID.start(advancedDeviceSignals: Boolean) {
             val keyService = NIDAdvKeyService()
             var fpjsRetryCount = 0
             val FPJS_RETRY_MAX = 3
+            var endpointUrl = Constants.fpjsDevDomain.displayName
+            if (NeuroID.getInstance()?.getEnvironment() == "LIVE"){
+                endpointUrl = Constants.fpjsProdDomain.displayName
+            }
```
Comment (kevinsites) [inline NeuroID/src/advancedDeviceAndroidLib/java/com/neuroid/tracker/extensions/AdvancedDeviceExtension.kt:32]: This is my fault for putting it in the ticket (I was making it quick). Although there is a dev option, we should not be putting that in the SDK as our customers regardless of their `env` will only ever call PROD. Only we as internal NID devs will call the dev endpoint.

**Review by kevinsites — CHANGES_REQUESTED** [review]

File: NeuroID/src/advancedDeviceAndroidLib/java/com/neuroid/tracker/extensions/AdvancedDeviceExtension.kt (hunk 1 above)
Comment (priya-xavier) [reply NeuroID/src/advancedDeviceAndroidLib/java/com/neuroid/tracker/extensions/AdvancedDeviceExtension.kt]: Ok. I removed the dev endpoint completely from the sdk.

Comment (priya-xavier) [reply NeuroID/src/advancedDeviceAndroidLib/java/com/neuroid/tracker/extensions/AdvancedDeviceExtension.kt]: Should we document the dev endpoint in the readme?

Comment (priya-xavier) [reply NeuroID/src/advancedDeviceAndroidLib/java/com/neuroid/tracker/extensions/AdvancedDeviceExtension.kt]: Adding the comment in the Jira ticket for documentation.

**Review by kevinsites — APPROVED** [review]

**Review by kishisaka-nid — APPROVED** [review]

File: NeuroID/src/advancedDeviceAndroidLib/java/com/neuroid/tracker/extensions/AdvancedDeviceExtension.kt (hunk 1 above)
Comment (priya-xavier) [reply NeuroID/src/advancedDeviceAndroidLib/java/com/neuroid/tracker/extensions/AdvancedDeviceExtension.kt]: Tested with site id and key for adv enabled form_1oLmFMjl and its working as expected.

**Pull request merged**